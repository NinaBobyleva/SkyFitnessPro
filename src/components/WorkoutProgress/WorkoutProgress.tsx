export function WorkoutProgress() {
  return (
    <div className="w-auto h-[6px] bg-grayLight  rounded-full ">
      {/* <div className="rounded-full h-[100%] bg-blueLight"></div> */}
    </div>
  );
}
