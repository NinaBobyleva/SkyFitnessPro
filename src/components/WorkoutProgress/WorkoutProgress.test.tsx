import renderer from "react-test-renderer";
import WorkoutProgress from "./WorkoutProgress";

describe("Компонент Header", () => {
  it("Успешный рендер Header", () => {
    const component = renderer
      .create(
        <WorkoutProgress title="округлите спину" progress="60%" />
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
