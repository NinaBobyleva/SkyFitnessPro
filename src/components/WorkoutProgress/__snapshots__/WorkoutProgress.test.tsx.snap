// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент Header Успешный рендер Header 1`] = `
<div>
  <p
    className="text-black text-[18px] font-roboto-400 font-normal mb-[10px]"
  >
    округлите спину 60%
  </p>
  <div
    className="w-auto h-[6px] bg-grayLight  rounded-full "
  >
    <div
      className="rounded-full h-[100%] bg-blueLight"
      style={
        {
          "width": "60%",
        }
      }
    />
  </div>
</div>
`;
